autogluon/task/tabular_prediction/hyperparameter_configs.py

```diff
@@ -0,0 +1,49 @@
+
+import copy
+
+
+# Dictionary of preset hyperparameter configurations.
+hyperparameter_config_dict = dict(
+    # Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
+    default={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {},
+        'XT': {},
+        'KNN': {},
+        'custom': ['GBM'],
+    },
+    # Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
+    light={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {'max_depth': 15},
+        'XT': {'max_depth': 15},
+        'custom': ['GBM'],
+    },
+    # Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
+    very_light={
+        'NN': {},
```

Comment:
Do we maybe want to make NN smaller in this one? I'm not sure whether it matters relative to the other models, but if we want to make NN smaller, hyperparams like the following:

'NN': {'max_layer_width': 512, 'max_embedding_dim': 25, 'embedding_size_factor': 0.25}

Reply:
Relative to the others it is fine as-is, RF with 15 max-depth still get to 60 MB often, whereas NN is at most a few MB.